Ntuplizer/samples/Summer16/samples_crab_data.txt

```diff
@@ -1,7 +1,3 @@
-/JetHT/Run2016B-PromptReco-v1/MINIAOD
```

Comment:
I heard that any of the runs in 2016B-v1 are actually certified in the golden json so for 2016B (all of the certified runs are in 2016B-v2)
But I might be wrong ? Do you have some page to see if this is the case or not ?